.travis/before_install.sh

```diff
@@ -78,19 +78,19 @@ fi
 git clone https://github.com/pulp/pulp-openapi-generator.git
 if [ -n "$PULP_BINDINGS_PR_NUMBER" ]; then
   cd pulp-openapi-generator
-  git fetch origin +refs/pull/$PULP_BINDINGS_PR_NUMBER/merge
-  git checkout FETCH_HEAD
+  git fetch origin pull/$PULP_BINDINGS_PR_NUMBER/head:$PULP_BINDINGS_PR_NUMBER
```

Comment:
Nice!

Reply:
You can thank @daviddavis for updating the templated before_install.sh.j2 with this fix:
https://pulp.plan.io/issues/5990

Reply:
@fabricio-aguiar I'm curious, were you getting broken by PRs that were already merged?

Reply:
not here, but I got it in some plugins in the past